spa/src/analytics/analytics-proxy-client.ts

```diff
@@ -0,0 +1,28 @@
+import { AnalyticClient, ScreenEventProps, TrackEventProps, UIEventProps } from "./types";
+import { axiosRest } from "../api/axiosInstance";
+import { reportError } from "../utils";
+const sendAnalytics = (eventType: string, eventProperties: Record<string, unknown>, eventAttributes?: Record<string, unknown>) => {
+	axiosRest.post(`/rest/app/cloud/analytics-proxy`,
+		{
+			eventType,
+			eventProperties,
+			eventAttributes
+		}).catch(reportError);
```

Comment:
![giphy (1)](https://github.com/atlassian/github-for-jira/assets/37155488/25a7bcac-4459-462a-8848-e290ea5a2329)